### ISSUE TITLE
play_store_key / manifest.json 

### ISSUE THREAD

**Issue by RegencySoftware** [AlexDisler/cordova-plugin-inapppurchase#29]
Is there any other way to specify or pass the play_store_key other than through manifest.json?

**Comment by RegencySoftware** [comment]
I found a link where other folks hit this error. When they passed as product ID the connection worked (but later failed after making the first purchase this is an open issue on another ticket)
. 
https://github.com/poiuytrez/AndroidInAppBilling/issues/93

I am now passing a product id on the init and it worked (well sort of) it crashes now after the first purchase. Closing this ticket and will reply to the ticket that has this issue.

inappbilling.init(successHandler, errorHandler, { showLog: true }, "my.product.id");
